docs/Getting Started/Install and Run/index.md

```diff
@@ -126,4 +127,68 @@ If Core was built with `EXTENDED_POLICY=EXTERNAL_PROPRIETARY`, the `core.sh` scr
 ```bash
 ./core.sh <command> false
 ``` 
-!!!
+## A cMake Exercise
+
+To Build Core using the HTTP Feature Support Option, do the following:
+
+### **NOTE:** If you have already built Core , be sure to remove the existing build by executing the following steps:
+
+```
+rm -rf sdl_core
+rm -rf build
+```
+## From the command line run:
+
+```
+sudo apt-get install git cmake build-essential sqlite3 libsqlite3-dev libssl-dev libssl1.1 libusb-1.0-0-dev libudev-dev libgtest-dev libbluetooth3 libbluetooth-dev bluez-tools libpulse-dev python3-pip python3-setuptools python
+```
+Execute This Line:
+
+```
+git clone https://github.com/smartdevicelink/sdl_core
+```
+Now, Execute This Line:
+
+```
+cd sdl_core
+```
+
+Execute This Line:
+
+```
+git checkout develop
+```
+Execut This Line:
+
+```
+git submodule init
+```
+
+Execute This Line:
+
+```
+git submodule update
+```
+
+Next Step is to Create BUILD folder **outside** of core build:
+cd..
+mkdir build
+cd build
+
+Execute This Line - This will build core using the HTTP option:
+
+```
+cmake ../sdl_core -DEXTENDED_POLICY=HTTP
+```
+
+Now, Execute This Line:
+
+```
+make -j3
+```
+And finally Execute this Line:
+
+```
+make install
+```
+You have successfully built Core using the HTTP Feature Support Option!
```

Comment:
With all of these suggestions, I would rewrite this section like this:

## Example - EXTERNAL_PROPRIETARY build

!!! Note
To perform a completely clean build after previously building SDL Core, delete the existing build folder before running these steps:
```
rm -rf sdl_build
```
!!!


The following set of commands can be used to build the develop branch of SDL Core from scratch with the `EXTERNAL_PROPRIETARY` policy mode enabled:

```
sudo apt-get install git cmake build-essential sqlite3 libsqlite3-dev libssl-dev libssl1.1 libusb-1.0-0-dev libudev-dev libgtest-dev libbluetooth3 libbluetooth-dev bluez-tools libpulse-dev python3-pip python3-setuptools python
git clone https://github.com/smartdevicelink/sdl_core
cd sdl_core
git checkout develop
git pull
git submodule init
git submodule update
cd ..
mkdir sdl_build
cd sdl_build
cmake ../sdl_core -DEXTENDED_POLICY=EXTERNAL_PROPRIETARY
make install-3rd_party
make -j3 install
```

Reply:
Agreed

Reply:
I just tried running the above steps and got an error saying sdl_core already exists...so we'd have to delete the sdl_core folder as well.

Reply:
Did that step failing cause the whole process to fail? If not, I don't think it's really necessary. The build should still finish, even if the folder already exists.

Reply:
Yep because I already have it installed.  So, to your point, it won't fail if its the FIRST TIME install.  But, if it is NOT, this step will fail for sure

Reply:
We could always separate out the first two commands as "first install only" if you want, but I don't see any reason to delete the `sdl_core` folder every single time

Reply:
@daishah alright, I've updated my [original comment](https://github.com/smartdevicelink/sdl_core_guides/pull/129/files#r679305650) to lay out the steps in a clearer manner.